### ISSUE TITLE
Horizontal Scroll on Pages

### ISSUE THREAD

**Issue by tfont** [cretz/doogie#60]
Ideally located on the bottom of pages for scrolling to the left and right on pages that ARE wider than the active window size.

**Comment by cretz** [comment]
I don't understand this bug. What websites are not showing horizontal scroll that do on other browsers when the browsers are the same size? Can you give an example? I have confirmed that Doogie does properly show horizontal scrollbars on the browser for wide pages.

**Comment by tfont** [comment]
Yes, when text is used within a div that is longer than the page horizontally.

I try to provide an example soon, if still necessary.

**Comment by cretz** [comment]
@tfont - definitely necessary. Screenshot too would help.
